qemu/tests/cfg/block_stream.cfg

```diff
@@ -108,3 +121,36 @@
                     when_streaming = "pause_job set_speed resume_job cancel start wait_for_finished"
                     Host_RHEL.m6:
                         when_streaming = "set_speed cancel start wait_for_finished"
+        - data_integrity:
+            type = block_stream_simple
+            before_start = "create_files stop"
+            after_finished = "resume verify_md5s"
+        - stop_cont:
+           type = block_stream_simple
+           before_start = "create_files"
+           when_streaming = "stop resume stop resume stop resume"
+           after_finished = "reboot verify_alive"
```

Comment:
Could you explain why create files for this case ?   and without verify md5.

Reply:
@yanan-fu Yes, just to make sure the block is not empty, to actually stream something, but no need to check integrity for this case.